### DISCUSSION TITLE
[Feature] search name

### DISCUSSION THREAD

**Discussion by romain1206** [immich-app/immich#5207] [Feature Request]
### The feature

hello,

Would it be possible to add a search function for people?

especially in the merge part when there are a lot of people

![2023-11-21 00 34 53](https://github.com/immich-app/immich/assets/16817502/304f3649-e097-4ed4-8811-b61d9dd16293)

many thanks for this software

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5209

**Reply by romain1206** [reply]
very good
it's wonderfull
